kernel/src/lib.rs

```diff
@@ -60,13 +60,11 @@ pub use table::Table;
 #[cfg(feature = "arrow-conversion")]
 pub mod arrow_conversion;
 
-#[cfg(feature = "simple-client")]
-pub mod simple_client;
-
-#[cfg(feature = "default-client")]
+#[cfg(any(feature = "default-client", feature = "sync-client"))]
 pub mod client;
+
 #[cfg(feature = "default-client")]
-pub use client::*;
+pub use client::default::*;
```

Comment:
Two questions:
1. Didn't `client` already `pub use default::*`? Why do we need to pull it again here?
2. Assuming we _do_ need to pull it here, any reason not to conditionally pull in the sync client as well?

Reply:
Solved by removing the star imports entirely